homeassistant/components/fritz/common.py

```diff
@@ -795,6 +800,15 @@ def get_wan_link_properties(self) -> dict[str, Any]:
             "WANCommonInterfaceConfig", "1", "GetCommonLinkProperties"
         )
 
+    def ipv6_active(self) -> bool:
+        """Call WANIPConn service."""
+
+        return bool(
+            self._service_call_action(
+                "WANIPConn", "1", "X_AVM_DE_GetExternalIPv6Address"
+            )["NewExternalIPv6Address"]
+        )
```

Comment:
we should use the already existing property `self.fritz_status.external_ipv6` - no need to do a "low level" call here